src/utils.ts

```diff
@@ -1,10 +1,8 @@
 import { PluginKey, Selection } from 'prosemirror-state';
 import { DecorationSet } from 'prosemirror-view';
 
-export const DEFAULT_ID = 'autocomplete';
-export const DEFAULT_DECO_ATTRS = { id: DEFAULT_ID, class: DEFAULT_ID };
-
-export const pluginKey = new PluginKey(DEFAULT_ID);
+export const AUTOCOMPLETE = 'autocomplete';
```

Comment:
Not quite sure what to call this. It is both the key name and the class name. Any thoughts?